### ISSUE TITLE
Export-Excel not exporting "real" dates

### ISSUE THREAD

**Issue by Ben-Conrad** [dfinke/ImportExcel#234]
Hello, this issue seems similar to #118.

When I export to CSV excel treats the cell as a date.  I'm using ToShortDateString() to keep things simple. 
 When I export-excel the same data Excel will not treat the cells as dates. This causes all sorts of problems with sorting, pivot tables, filters, etc.

**# To Reproduce**
touch file1.txt file2.txt file3.txt file4.txt file5.txt file6.txt file7.txt file8.txt file9.txt file10.txt

(gci .\file1.txt).LastWriteTime = '1/1/2017'
(gci .\file2.txt).LastWriteTime = '2/1/2017'
(gci .\file3.txt).LastWriteTime = '3/1/2017'
(gci .\file4.txt).LastWriteTime = '4/1/2017'
(gci .\file5.txt).LastWriteTime = '5/1/2017'
(gci .\file6.txt).LastWriteTime = '6/1/2017'
(gci .\file7.txt).LastWriteTime = '7/1/2017'
(gci .\file8.txt).LastWriteTime = '8/1/2017'
(gci .\file9.txt).LastWriteTime = '9/1/2017'
(gci .\file10.txt).LastWriteTime = '10/1/2017'

del test.csv -erroraction SilentlyContinue
del test.xlsx -erroraction SilentlyContinue
del test2.xlsx -erroraction SilentlyContinue

**# Create the CSV, dates look fine**
gci | select Name, @{name = "LastWriteTime"; expression = {$_.LastWriteTime.ToShortDateString() } } | export-csv -NoTypeInformation test.csv

![image](https://user-images.githubusercontent.com/11560403/31728757-2671a6b0-b3fb-11e7-9550-3ae054d5adab.png)

**Send the CSV to EXCEL, this is where I see an issue**
import-csv test.csv | export-excel -Path test.xlsx
 
![image](https://user-images.githubusercontent.com/11560403/31728771-2fecf898-b3fb-11e7-89ab-79222663c029.png)

**Bypassing the export and import from CSV has the same result (same problem):**
gci | select Name, @{name = "LastWriteTime"; expression = {$_.LastWriteTime.ToShortDateString() } } | export-excel -Path test2.xlsx

Is there a fix for this?

**Comment by Ben-Conrad** [comment]
Version 4.0.2:

PS C:\> (get-module importexcel).version

Major  Minor  Build  Revision
-----  -----  -----  --------
4      0      2      -1

**Comment by DarkLite1** [comment]
Did some testing and according to me it works as designed By this I mean that when `Export-Excel` receives an object of `DateTime`, it will accordingly store this as a `DateTime` field in Excel. When the date is formatted to a `String` with a method like `ToShortDateString()`, the object is no longer of the type `DateTime` and is, as such, stored as a `String` in Excel.

An example to demonstrate what I mean:

```
$Params = @{
    Path     = 'Test.xlsx'
    AutoSize = $true
}

Remove-Item  $Params.Path -EA Ignore

Get-ChildItem | Select-Object -First 2 -Property Name, 
@{N = 'LastWriteTimeToString'; E = {$_.LastWriteTime.ToShortDateString() }},
@{N = 'LastWriteTimeOriginal'; E = {$_.LastWriteTime }},
LastWriteTime | Export-Excel -@Params -Verbose
```

So in short, the current design mandates a `DateTime` object if you wish to have a `DateTime` field in Excel.

**Comment by Ben-Conrad** [comment]
Thank you for looking into this.  The -verbose switch is helpful.  I also found that since I am importing from a .CSV that I have to cast the fields properly before sending them to export-excel.  Similar to [this ](https://stackoverflow.com/questions/10585289/casting-column-from-csv-as-date-powershell ) link.

**Comment by DarkLite1** [comment]
@dfinke I knew there would a point where people would appreciate the extended Verbose messages :wink: 

Glad it's solved for you @Ben-Conrad :+1:
